main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -165,6 +165,7 @@ class Flix {
     "SemiGroup.flix" -> LocalResource.get("/src/library/SemiGroup.flix"),
     "CommutativeSemiGroup.flix" -> LocalResource.get("/src/library/CommutativeSemiGroup.flix"),
     "Monoid.flix" -> LocalResource.get("/src/library/Monoid.flix"),
+    "CommutativeMonoid.flix" -> LocalResource.get("/src/library/CommutativeMonoid.flix"),
```

Comment:
This list is getting long. Can I kindly ask you to sort it alphabetically? :)

Reply:
Yeah sure :)

Reply:
Do you want the entire list sorted or just the TCs?